manifests/gcp_marketplace/deployer/Dockerfile

```diff
@@ -1 +1 @@
-FROM gcr.io/ml-pipeline/marketplace-k8s-app-tools/k8s/deployer_helm/onbuild:remove-ui-ownerrefs
\ No newline at end of file
+FROM gcr.io/cloud-marketplace-tools/k8s/deployer_helm/onbuild:latest
```

Comment:
Once you figured out a image that works, could you ping the image version instead of using latest?

Reply:
I'm following the guide to setup better release process.
For each release, we will rebuild Deployer, put to QA, use corresponding latest verify tool etc.

I will add Erika to review after #2391. Feel free to ignore this PR for now.